app/static/js/cluster.js

```diff
@@ -102,6 +102,17 @@ fetch('js/data.json', {mode: 'no-cors'})
 
   var png64 = cy.png(img_options);
 
+  /*
+  var mol = data.nodes[0].data.id;
+  let smilesDrawer = new SmilesDrawer.Drawer(options);
+  SmilesDrawer.parse(mol, function(tree) {
+    // Draw to the canvas
+    smilesDrawer.draw(tree, "drawing", "light", false);
+    // Alternatively, draw to SVG:
+    // svgDrawer.draw(tree, 'output-svg', 'dark', false);
+  });
+  */
```

Comment:
Uncomment this section and then run to test that the mol draws

Reply:
Looks good! should we uncomment before merging?

Reply:
Nope! I'm leaving it like this until @klu17 gets her popper pr merged that way we can get the drawing integrated into those